docs/destinations/forwarding-events/snowbridge/configuration/transformations/builtin/base64Decode.md

```diff
@@ -0,0 +1,13 @@
+# base64Decode
+
+`base64Decode`: Base64 decodes the message's data.
```

Comment:
State that it’s available since version X.Y.Z. Also, have you updated the componentVersions file?